_data/releases.yml

```diff
@@ -0,0 +1,3346 @@
+- version: beta-20160407
+  major_version: v1.0
+  release_date: '2016-04-07'
+  release_type: Testing
+  go_version: go1.8.3
+  sha: 96f4c50a69272cfd5a32e65889f5f72033502ed2
+  docker:
+    docker_image: cockroachdb/cockroach-unstable
+- version: beta-20160414
```

Comment:
Please add a blank line between each version for readability. YAML should ignore it.

For maintainability, would it be better to have a YAML file per major version that contains all the minor releases in that version? Or would that be too hard for the release-note script to accommodate?

Reply:
Done. For now, it would be better to keep all in one file, but depending on how branch-based versioning goes, we may end up needing to split out each YML file into its own, per-version file. I should be able to make that work in the script, though.

Reply:
I'm not seeing the blank lines between versions...